From ff1c90f70127dec21d5cb8ded5206f286500f7f7 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Tue, 26 Nov 2019 11:30:15 +0800
Subject: [PATCH] ARM: dts: rockchip: add rk3228 vop compatible name for rk322x

Upstream kernel and uboot use rockchip,rk3228-vop as rk322x vop
compatible name, so add this.

Fixes: 810b044867ee ("ARM: dts: rk322x: add vop display node")
Change-Id: I552b99a64f8de679aa4504624e35482e2b0a0508
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 arch/arm/boot/dts/rk322x.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk322x.dtsi b/arch/arm/boot/dts/rk322x.dtsi
index f52489c1cb47..427d3ada666d 100644
--- a/arch/arm/boot/dts/rk322x.dtsi
+++ b/arch/arm/boot/dts/rk322x.dtsi
@@ -937,7 +937,7 @@
 	};
 
 	vop: vop@20050000 {
-		compatible = "rockchip,rk322x-vop";
+		compatible = "rockchip,rk3228-vop", "rockchip,rk322x-vop";
 		reg = <0x20050000 0x1ffc>;
 		reg-names = "regs";
 		interrupts = <GIC_SPI 32 IRQ_TYPE_LEVEL_HIGH>;
-- 
2.35.3

